### ISSUE TITLE
how can I install GUROBI pod in my pod collection?

### ISSUE THREAD

**Issue by robodreamer** [RobotLocomotion/drake#184]
Hi, I was just trying out installing drake and running the tests. 
How can I add GUROBI pod in my pod collection? Maybe I am not familiar with pod collection so I don't know what to do with this warning below:

 GUROBI MEX not found.  GUROBI MEX support will be disabled.
    To enable, install the GUROBI pod in your pod collection, and rerun make config; make in drake

Thanks!

**Comment by brandonjdehart** [comment]
For others' reference, in a fresh Ubuntu 14.04.1 installation I also needed to "sudo apt-get install libwww-perl libterm-readkey-perl" to make the gurobi authentication script work.

**Comment by RussTedrake** [comment]
i'll add it to the install prereqs.  thanks.

**Comment by RussTedrake** [comment]
actually - those were already in the install_prereqs.sh script.

**Comment by brandonjdehart** [comment]
My bad, I installed the prereqs from drake-distro before I cloned the gurobi files.

I also ended up having to copy the pods.cmake and matlab_pods.cmake files from the spotless pod into the gurobi cmake folder, and modified the CMakeLists.txt to call the pod "gurobi" instead of "drake".

**Comment by RussTedrake** [comment]
I apologize.  The instructions could have been more clear.  You needed to checkout recursive to get the contents of the cmake folder. 

I think this would have been better:

```
cd drake-distro
git clone https://github.com/RobotLocomotion/gurobi --recurse-submodules
cd gurobi
sudo ./install_prereqs.sh ubuntu   # change this for your platform
make
cd ../drake
make configure
make
```
and optionally add `gurobi` to `drake/distro/tobuild.txt` (before the entry for drake).

**Comment by brandonjdehart** [comment]
Thanks! The updated instructions work well.

**Comment by jackchenxiaojiao** [comment]
Hi,  Russ,
I get error when log into gurobi.
I checked the gurobi website, and found  the website had changed .
So I changed the file "download_gurobi.pl" 
 from 
       $response = $mech->get( "http://www.gurobi.com/ACCOUNT" );   (line27) 
       ...
       $response = $mech->get( "http://WWW.gurobi.com/download/gurobi-optimizer" );   (line38)
 to 
         $response = $mech->get( "http://www.gurobi.com/LOGIN" );  
       ...
       $response = $mech->get( "http://USER.gurobi.com/download/gurobi-optimizer" );

But there are still some problems.
Can u help me?

**Comment by RussTedrake** [comment]
Our build servers download from the gurobi website every day - so i don't *think* the links need to change?

If you can paste in the specific error i will try to help.

- Russ

**Comment by brandonjdehart** [comment]
It looks like as of this morning both the gurobi website login process and the available versions have changed (they provide 5.6.3 now but not 5.6.2). That being said, even a make clean in the gurobi folder doesn't erase the existing library, so as long as you download the files manually it works just fine from then on.

**Comment by RussTedrake** [comment]
Awesome.  Thanks Brandon.  I'll update the pod tonight.

- Russ

**Comment by jackchenxiaojiao** [comment]
Hi Russ, I think I have solved my problem with gurobi.
First, as previously I said, I changed the file "download_gurobi.pl".
        from "http://www.gurobi.com/account"       (line27)
        to     "http://www.gurobi.com/login"
        from "http://www.gurobi.com/download/gurobi-optimizer"       (line38)
        to     "http://user.gurobi.com/download/gurobi-optimizer" )
Second, as Brandon mentioned, I downloaded the gurobi 5.6.3 manually, and placed it  as drake-distro/gurobi/gurobi5.6.3_linux64.tar.gz.
Third, I replaced all the "562" with "563" and "5.6.2" with "5.6.3" in  the makefile and CMakeLists.txt in drake-distro/gurobi. 
Then it can make now.
ALso, in the building process I need to pre-install other pods like eigen and bullet.
Now I have succesfully built the file QPcontroller.mex.
Thanks Russ and Brandon.
